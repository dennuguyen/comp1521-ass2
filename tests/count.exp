<stdin>:18: invalid instruction code: 0401FFF7
