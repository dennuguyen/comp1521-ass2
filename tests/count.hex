3c011001
34240000
34020004
c
34020005
c
24020
34090001
109082a
14200009
92020
34020001
c
3404000a
3402000b
c
21290001
401fff7