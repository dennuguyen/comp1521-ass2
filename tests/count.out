Program
  0: lui  $1, 4097
  1: ori  $4, $1, 0
  2: ori  $2, $0, 4
  3: syscall
  4: ori  $2, $0, 5
  5: syscall
  6: add  $8, $0, $2
  7: ori  $9, $0, 1
  8: slt  $1, $8, $9
  9: bne  $1, $0, 9
 10: add  $4, $0, $9
 11: ori  $2, $0, 1
 12: syscall
 13: ori  $4, $0, 10
 14: ori  $2, $0, 11
 15: syscall
 16: addi $9, $9, 1
 17: bgez $1, $0, -9
Output
Unknown system call: 4
Registers After Execution
$1  = 268500992
$2  = 4
$4  = 268500992
